youtube/youtube.go

```diff
@@ -212,7 +223,7 @@ func (o *YouTube) Grab(url string, options *Options) (ret *VideoInfo, err error)
 	}
 
 	if options.Transcript {
-		if ret.Transcript, err = o.GrabTranscript(videoId); err != nil {
+		if ret.Transcript, err = o.GrabTranscript(videoId, "en"); err != nil {
```

Comment:
I didn't see where this was used and I believe this might have been left over from the previous standalone version.